<ui version="4.0" >
 <class>XWidgetEditFile</class>
 <widget class="QWidget" name="XWidgetEditFile" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>File Path</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLineEdit" name="filePath" />
      </item>
      <item>
       <widget class="QPushButton" name="selectButton" >
        <property name="text" >
         <string>Select File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
